.circleci/config.yml

```diff
@@ -111,26 +120,23 @@ jobs:
             pip install nox
             pip install dataclasses
             nox -s lint test_tools test_core test_jupyter_notebooks -ts
-
-
   test_linux:
     parameters:
       py_version:
         type: string
     docker:
-      - image: circleci/python:<< parameters.py_version >>
+      - image: circleci/python:3.8
```

Comment:
at this point we should probably use a vanilla ubuntu image and not a Python one, right?

Reply:
it seems like they don't have 18.04, only either 20.04 (beta) or 16.04 (https://circleci.com/docs/2.0/configuration-reference/#available-machine-images)

we could use  `cimg/base:stable-18.04` docker image as an alternative (https://discuss.circleci.com/t/ubuntu-20-04-next-gen-convenience-images/36604)

Reply:
Interesting. Let's try ubuntu-16.04 until the 20.04 comes out of beta?

Reply:
Sure. I'm curious why you prefer the machine to docker? Any specific concern?

Reply:
Nothing major. less moving parts.
If you want to try a docker image I am fine with it, but I am not sure what the implications are.
Pay attention to startup and runtime speeds if you do.

Reply:
My initial wrong perception was that the cimg image is a third party one.

Reply:
ah i see, yea, the docker image is from circleCI :)